pkg/config/validation_test.go

```diff
@@ -236,7 +236,104 @@ func TestValidateFromHostSyncMappings(t *testing.T) {
 				Enabled: true,
 				Selector: config.FromHostSelector{
 					Mappings: map[string]string{
-						"default/my-cm": "barfoo",
+						"default/*": "barfoo",
+					},
+				},
+			},
+			expectErr: expectErr,
+		},
+		{
+			name: "invalid config 5",
```

Comment:
nit: Would be nice if the names described what was invalid like "mappings has invalid key"

Reply:
changed :+1: